chromium_src/chrome/browser/ui/browser_navigator.cc

```diff
@@ -32,7 +32,7 @@ bool IsURLAllowedInIncognitoBraveImpl(const GURL& url) {
 
   if (host == kRewardsPageHost || host == chrome::kChromeUISyncInternalsHost ||
       host == chrome::kChromeUISyncHost || host == kAdblockHost ||
-      host == kWelcomeHost) {
+      host == kWelcomeHost || host == kBraveGettingStartedHost) {
```

Comment:
why aren't these allowed in incognito  (this and the welcome page)? It's not clear to me why it would be a problem @diracdeltas ?

Reply:
i guess some of the commands (open leo/wallet/rewards) are for stuff that's not default enabled in private windows so if there's no open regular window we'd have to open one

Reply:
if this is going to be allowed in incognito we need to make sure the server checker's requests go through Tor when the welcome page is loaded in a Tor window